Visualizer/pipper.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import sys
+
+
+def install_packages(packages: list):
```

Comment:
More my opinion, I don't think this should be up to us to configure. We should request the server maintenance to install the python packages we want.

Reply:
Asked the code review group, conclusion is to remove it for now